docs/cookbook.rst

```diff
@@ -249,3 +249,55 @@ Unless of course something bad happens inside of ``PerformA`` in which case the
     AssertionError:
     Expected: <True>
          but: was <False>
+
+
+Using Either
+===========
+
+Sometimes you may need to use a try/except control flow in your test, for 
+one reason or another. Luckily, your Actor can perform this flow 
+with the :class:`~screenpy.actions.Either` Action!::
+
+    the_actor.will(Either(DoAction()).or_(DoDifferentAction())
+
+The Actor will attempt to perform the first action (or set of actions).  
+If successful, the Actor moves on. but an `AssertionError` is raised the Actor will 
+begin performing the second action (or set of actions) passed into :meth:`~screenpy.actions.Either.or_`.
+Note the Actor only catches `AssertionError` here allowing for other exceptions to still be raised.
+
+:class:`~screenpy.actions.Either` allows users to pass in multiple actions. This is 
+similar Actor performing multiple actions in one call::
+
+    the_actor.will(
+        Either(
+            DoAction1(),
+            DoAction2(),
+            DoAction3(),
+        ).or_(
+            DoDifferentAction1(),
+            DoDifferentAction2(),
+            DoDifferentAction3(),
+        )
+    )
+
+
+.. note::
+   :class:`~screenpy.actions.Either` will not describe any cleanup for the Actor
+   after it experiences a failure in the first routine; the Actor will proceed directly
+   to the second routine. Keep this in mind while defining the two branches of Actions.
+
+To help illustrate this further here is a real-world example using screenpy_selenium::
+
+    the_actor.will(
+       Either(
+           See(BrowserURL(), HasUrlNetloc(URL)),
```

Comment:
We dont' have a `HasUrlNetloc` Resolution, but we do have `ReadsExactly`:
```python
See(BrowserURL(), ReadsExactly(URL))
```

or `Matches`:
```python
See(BrowserURL(), Matches(r"^.*/screenpy-docs\..*$"))
```

Reply:
Woops!  I used my own action in the example.  Shoot.  it's one that I have yet to submit for screenpy_selenium.  
I'll change the example.